move_base/src/move_base.cpp

```diff
@@ -1181,4 +1181,46 @@ namespace move_base {
       controller_costmap_ros_->stop();
     }
   }
+
+  bool MoveBase::getRobotPose(tf::Stamped<tf::Pose>& global_pose, costmap_2d::Costmap2DROS* costmap)
```

Comment:
btw @mikeferguson, do you want me to apply the requested changes? Or will you do by yourself?